hexrdgui/calibration/polarview.py

```diff
@@ -401,14 +401,13 @@ def apply_image_processing(self):
         # masks should also be distorted as well.
 
         # We only apply "visible" masks to the display image
-        img = self.apply_visible_masks(img)
-        disp_img = self.apply_tth_distortion(img)
+        img = self.apply_tth_distortion(img)
+        disp_img = self.apply_visible_masks(img)
         self.display_image = disp_img
 
         # Both "visible" and "boundary" masks are applied to the
         # computational image
         comp_img = self.apply_boundary_masks(img)
```

Comment:
@bnmajor For the `comp_img`, we want to apply *both* the visible masks and the boundary masks, right? So should the input to this function be the `disp_img` (where the visible masks have been applied)?

Reply:
Same for `reapply_masks`

Reply:
I believe this will fix #1690

Reply:
Sounds good! Fixed now.